index.bs

```diff
@@ -931,5 +931,103 @@ partial interface HTMLIFrameElement {
 </section>
 <section id="privacy" class="informative">
   <h2 id="privacy-and-security">Privacy and Security</h2>
-  <p class="issue">TODO</p>
+  <p>This specification standardizes a mechanism for an embedding page to set a
+  policy which will be enforced on an embedded page. Similar to iframe
+  <{iframe/sandbox}>, this can be done without the express permission of the
+  embedded page, which means that behaviors of existing features can be changed
+  in published web sites, by embedding them in another document with an
+  appropriate container policy.</p>
+
+  <p>As such, the biggest privacy and security concerns are:</p>
+  <ul>
+    <li>Exposure of behavior in a cross-origin subframe to its embedder</li>
```

Comment:
I don't know the styles. But should we link these 2 bullet points with 10.1 and 10.2?